### PR TITLE
BUG fixed: mask tensor should be on cuda also by condition.

### PR DESCRIPTION
**Pull request by zhuobinggang**
BUG fixed: mask tensor should be on cuda also by condition.

I was using this repository, and there was a ERROR when using the GPU, and I changed the source code of the local repository myself and it worked.

### CODE REVIEW

**Comment by kmkurn** [comment]
Hi, thanks for the PR. About your issue, have you checked that `tags` has been moved to GPU? Tensor `mask` is created in the same device as `tags` because `torch.ones_like` is used.

**Comment by zhuobinggang** [comment]
Thank you, that is my problem.
